release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -0,0 +1,1627 @@
+==== Windows containers
+
+[id="ocp-4-19-technology-preview-tables_{context}"]
+== Technology Preview features status
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use. Note the following scope of support on the Red{nbsp}Hat Customer Portal for these features:
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the following tables, features are marked with the following statuses:
+
+* _Not Available_
+* _Technology Preview_
+* _General Availability_
+* _Deprecated_
+* _Removed_
+
+
+[discrete]
+[id="ocp-release-notes-auth-tech-preview_{context}"]
+=== Authentication and authorization Technology Preview features
+
+.Authentication and authorization Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.17 |4.18 |4.19
+
+|Pod security admission restricted enforcement
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|====
+
+[discrete]
+[id="ocp-release-notesedge-computing-tp-features_{context}"]
+=== Edge computing Technology Preview features
+
+.Edge computing Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.17 |4.18 |4.19
+
+|Accelerated provisioning of {ztp}
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Enabling disk encryption with TPM and PCR protection
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|====
+
+[discrete]
+[id="ocp-release-notes-installing-tech-preview_{context}"]
+=== Installation Technology Preview features
+
+.Installation Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.17 |4.18 |4.19
+
+// All GA in 4.17 notes for oci-first
+|Adding kernel modules to nodes with kvc
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Enabling NIC partitioning for SR-IOV devices
+|General Availability
+|General Availability
+|General Availability
+
+|User-defined labels and tags for {gcp-first}
+|General Availability
+|General Availability
+|General Availability
+
+|Installing a cluster on Alibaba Cloud by using Assisted Installer
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Mount shared entitlements in BuildConfigs in RHEL
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|{product-title} on {oci-first}
+|General Availability
+|General Availability
+|General Availability
+
+|Selectable Cluster Inventory
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Installing a cluster on {gcp-short} using the Cluster API implementation
+|General Availability
+|General Availability
+|General Availability
+
+|{product-title} on Oracle Compute Cloud@Customer (C3)
+|Not Available
+|General Availability
+|General Availability
+
+|{product-title} on Oracle Private Cloud Appliance (PCA)
+|Not Available
+|General Availability
+|General Availability
```

Comment:
These should be removed - they should have been feature blurbs in 4.18 (I think Brendan was going to update accordingly after 4.18 GA)
```suggestion
```